### PR TITLE
Solution of 7의 개수

### PR DESCRIPTION
**Pull request by sjh-judeee3**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (juyoungml) [inline .gitignore:1]: LGTM but consider using .gitignore templates.

File: README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ This repository is inspired from [programmers-coding-test](https://github.com/co
 
 | 번호 | 문제 출처 | 풀이 |
 | --- | ------- | --- |
+| 1 | [7의 개수](https://school.programmers.co.kr/learn/courses/30/lessons/120912) | [7의-개수.py](https://github.com/scottsuk0306/programmers/blob/main/level-0/7의-개수.py) |
```
Comment (juyoungml) [inline README.md:14]: 풀이 link has to be in the form of `https://github.com/scottsuk0306/programmers/blob/main/level-{problem_level}/{problem_title}.py`. Well done!

File: level-0/7의-개수.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+def solution(array):
+    answer = 0
+    
+    for num in array:
+        num = list(str(num))
+        answer = answer + num.count('7')
```
Comment (juyoungml) [inline level-0/7의-개수.py:6]: What's the time complexity for `count`?

Comment (sjh-judeee3) [reply level-0/7의-개수.py:6]: Time complexity for `count` is O(n).

**Review by juyoungml — APPROVED** [review]
LGTM! Keep up the good work!

**Pull request merged**